config/locales/blacklight.ar.yml

```diff
@@ -170,10 +178,18 @@ ar:
         no_items_found: 'لم يتم العثور على %{entry_name}'
         single_item_found: 'تم العثور على <strong>1</strong> %{entry_name}'
         pages:
+          zero: 'TODO'
           one: '<strong>%{start_num}</strong> - <strong>%{end_num}</strong> من <strong>%{total_num}</strong>'
+          two: 'TODO'
+          few: 'TODO'
+          many: 'TODO'
           other: '<strong>%{start_num}</strong> - <strong>%{end_num}</strong> من<strong>%{total_num}</strong>'
       entry_pagination_info:
+        zero: 'TODO'
```

Comment:
Relevant keys in English: 
https://github.com/projectblacklight/blacklight/blob/f401bdf1966af5d7e33f5f0e89e6b3665af42b95/config/locales/blacklight.en.yml#L175-L177